src/layout/Flex.js

```diff
@@ -1,6 +1,33 @@
-import React from 'react'
+import React, { Children, cloneElement } from 'react'
 
-// TODO: align items/content/justify/etc support
-export let Flex = ({ style, ...props }) => (
-  <div style={{ display: 'flex', ...style }} {...props} />
+export let Flex = ({
+  as: Component = 'div',
+  style,
+  styleItems,
```

Comment:
I'm not sure about this hack. I included it to see what you thought @daedalus28. The correct solution is to use CSS-in-JS and do `& > *` to style the children.